### PR TITLE
Thank everyone

### PR DESCRIPTION
**Pull request by hzoo**

### CODE REVIEW

**Comment by zxqfox** [comment]
Travis disagree with joining 😈

**Comment by hzoo** [comment]
It's fine 😄 (cuze I canceled the build and forgot to put ci skip)

File: README.md [hunk 1]
```diff
@@ -4,6 +4,15 @@
 
 Check out the [JSCS Compatibility](https://github.com/eslint/eslint/milestones/JSCS%20Compatibility) milestone to track the progress of [transitioning](http://eslint.org/blog/2016/04/welcoming-jscs-to-eslint) JSCS users to ESLint.
 
+The JSCS team would like to thank everyone for using JSCS and hope you'll be joining us with ESLint!
```
Comment (markelog) [inline README.md]: Seems duplicative we have these names already

Comment (hzoo) [inline README.md]: Ok I guess we could move it up

**Pull request closed without merging**